website/tests/a11y-helper.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+export const globalAxeOptions = {
+  rules: {
+    'color-contrast': { enabled: false },
```

Comment:
This should only be turned off in specific instances, not globally.

Reply:
@MelSumner this is how they were set before these changes:
https://github.com/hashicorp/design-system/pull/1823/files#diff-45093a48f9c59f17a4adbb1294bb86a6101e20105e5bd59fac1a5a36f293940b

maybe the declarations didn't get applied? 

In any case, I'll open a PR to remove these from the global declaration.

Reply:
https://github.com/hashicorp/design-system/pull/1833